libs/model/src/tester/bootstrap.ts

```diff
@@ -71,20 +72,25 @@ const migrate_db = async (sequelize: Sequelize) => {
 };
 
 /**
- * Truncates all tables
+ * Truncates all sequelize tables.
+ * Use with caution since this can cause deadlocks.
  * @param db database models
  */
 const truncate_db = async (db?: DB) => {
   if (!db) return;
-  try {
-    const tables = Object.values(db.sequelize.models)
-      .map((model) => `"${model.tableName}"`)
-      .join(',');
-    await db.sequelize.query(
-      `TRUNCATE TABLE ${tables} RESTART IDENTITY CASCADE;`,
-    );
-  } catch {
-    // ignore failed truncate
+  for (let i = 1; i <= 10; i++) {
```

Comment:
seems a little hacky if we need to retry the truncation 😅

Reply:
this was me trying to find the deadlocks,  but it. won't hurt